pkg/provisioner/cloudprovisioner.go

```diff
@@ -305,7 +305,7 @@ func (c *CloudProvisioner) CreateVolume(
 			err = status.Error(codes.NotFound, err.Error())
 			return nil, err
 		}
-		return nil, err
+		return nil, status.Error(codes.Internal, err.Error())
```

Comment:
Also need to update the error on Line 179 above, as it is done for controllerserver.go at line 113.

Reply:
Done